rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
```

Comment:
- [ ] Your initialize method should look similar to the below method :+1: to make associations properly :blush:

```ruby
  def initialize(date, book, person)
    @book = book
    book.rentals << self
    @person = person
    person.rentals << self
  end
```